iam/policy-templates/ci-cd.json

```diff
@@ -97,8 +97,8 @@
       ],
       "Resource": [
         "arn:aws:lambda:*:$account_id:function:dss-*",
-        "arn:aws:es:*:$account_id:domain/dss-index-dev",
-        "arn:aws:es:*:$account_id:domain/dss-index-dev/*",
+        "arn:aws:es:*:$account_id:domain/dss-*",
```

Comment:
Is this desirable?

Reply:
I think yes, because we want CI/CD to be able to deploy any of the stages.

Reply:
If so, then we probably need to fix the other bucket-specific/stage-specific perms.

Reply:
Backed out changes to this file, since CI/CD IAM perms are outside the scope of this PR. let's address those separately